app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java

```diff
@@ -49,6 +49,10 @@ public TestRunBroadRequestReceiver(PreferenceManager preferenceManager, EventLis
     public void onReceive(Context context, Intent intent) {
         String key = intent.getStringExtra("key");
         String value = intent.getStringExtra("value");
+        // If either key is null, do nothing
```

Comment:
Here you say "either": which other key should we be checking below?

Reply:
None.